spacy/pipeline/spancat.py

```diff
@@ -272,6 +272,24 @@ def predict(self, docs: Iterable[Doc]):
         scores = self.model.predict((docs, indices))  # type: ignore
         return indices, scores
 
+    def suggest_candidates(
```

Comment:
Should we call this `store_candidates` instead? The current naming sounds more like you would run `self.suggester` and return the results?